packages/beta/src/__tests__/api/documentsApi.unit.spec.ts

```diff
@@ -237,4 +237,53 @@ describe('Documents unit test', () => {
       .reply(200, { items: [] });
     await client.documents.feedback.reject([{ id: 1 }]);
   });
+
+  test('document preview uri', async () => {
+    const base = (path: string): string => {
+      return `/api/v1/projects/${project}/documents${path}`;
+    };
+
+    const prev = client.documents.preview;
```

Comment:
nit: `prev` usually reads `previous`. I'd prefer if you rename this to `preview`